docs/src/main/asciidoc/security-openid-connect.adoc

```diff
@@ -8,15 +8,17 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 include::./attributes.adoc[]
 :toc:
 
-This guide demonstrates how to use Quarkus OpenID Connect (OIDC) Extension to protect your JAX-RS applications using Bearer Token Authorization where Bearer Tokens are issued by OpenID Connect and OAuth 2.0 compliant Authorization Servers such as https://www.keycloak.org[Keycloak].
+You can use the Quarkus OpenID Connect (OIDC) extension to secure your JAX-RS applications using Bearer Token Authorization. 
+The Bearer Tokens are issued by OIDC and OAuth 2.0 compliant authorization servers, such as https://www.keycloak.org[Keycloak]. 
```

Comment:
@hmanwani-rh 
Capitalization of "Tokens" here. I think this should be a lower case "t" in this instance as it is a more generic reference. Also, in the phrase "Bearer Token authorization", "authorization" is in lower case

Reply:
@hmanwani-rh : The title doesn't appear to be changed here. Do you want to modify the title in this phase per our style guidelines?

Reply:
@hmanwani-rh @sheilamjones If we are OK with `Authorization Code Flow` then indeed, I guess it makes sense to keep it consistent, `Bearer Token Authorization` is a concept, a single piece... I know we have in some places  `Bearer token authorization` - may be we should do a dedicated review of all these expressions in another PR...

Reply:
Thanks @sberyozkin